lib/gui/app.js

```diff
@@ -198,6 +198,11 @@ app.config(($provide) => {
   });
 });
 
+app.config(($qProvider) => {
+  // Prevent errors generated by $uibModalInstance.dismiss() unhandled rejections
+  $qProvider.errorOnUnhandledRejections(false);
```

Comment:
Do you also get those transition errors when you start the application without setting this?

<img width="532" alt="screenshot 2017-02-06 15 29 08" src="https://cloud.githubusercontent.com/assets/2192773/22662897/0bbce6be-ec81-11e6-9df2-8e9d4f61d7f9.png">

That might be hiding yet another issue.